_data/email.yml

```
@@ -99,6 +99,15 @@ websites:
       img: migadu.png
       tfa: No
 
+    - name: Mimecast
+      url: https://www.mimecast.com/
+      img: mimecast.png
+      tfa: Yes
+      sms: Yes
+      email: Yes
+      software: Yes
+      doc: https://community.mimecast.com/docs/DOC-1913
```

I think this page is better. It describes how two enable 2FA. Something that the other page doesn't do.
```suggestion
      doc: https://community.mimecast.com/docs/DOC-1915
```